queries/cpon/highlights.scm

```diff
@@ -0,0 +1,45 @@
+; Literals
+
+(string) @string
+(escape_sequence) @string.escape
+
+(hex_blob
+  "x" @symbol
+  (_) @string)
+
+(esc_blob
+  "b" @symbol
+  (_) @string)
+
+(datetime
+  "d" @symbol
+  (_) @string.special)
```

Comment:
Maybe `@character.special` instead of `@symbol`?

Reply:
yeah that's fine with me